### DISCUSSION TITLE
[Question] Verification fails at 69%

### DISCUSSION THREAD

**Discussion by feoranis26** [luc-github/Repetier-Firmware-0.92#376] [Q&A] [answered]
I uploaded this firmware properly to my printer a few times before but when I tried to update it now it just gets stuck at verification at 69%

**Comment by luc-github** [comment, marked as answer]
https://github.com/luc-github/Repetier-Firmware-4-Davinci/discussions/185

**Comment by feoranis26** [comment]
I uploaded successfully a few times previously today, I didn't change anything with my setup between flashes. USB communication is good.

**Comment by luc-github** [comment]
cable close to light / or any power supply may disturb, also be sure upload speed is not too fast etc.. 
This part is definitly hardware / arduino related

**Comment by dpjpandone** [comment]
I has same problem using any version other than 1.5.8 (arduino ide)
